cypress/e2e/timeline/timeline.spec.ts

```diff
@@ -75,6 +75,35 @@ describe("Timeline", () => {
     let oldAvatarUrl: string;
     let newAvatarUrl: string;
 
+    /**
+     * Visit a room in the given layout.
+     * @param layout The layout name in which the room is opened. The name
+     * must be either IRC, Group, or Bubble.
+     */
+    const viewRoomByLayout = (layout: string) => {
+        if (layout == "IRC") {
+            // Enable IRC layout
+            cy.setSettingValue("layout", null, SettingLevel.DEVICE, Layout.IRC);
+        } else if (layout == "Group") {
+            // Enable modern layout. Traditionally modern layout has been specified with "Group"
+            cy.setSettingValue("layout", null, SettingLevel.DEVICE, Layout.Group);
+        } else if (layout == "Bubble") {
+            // Enable bubble message layout
+            cy.setSettingValue("layout", null, SettingLevel.DEVICE, Layout.Bubble);
+        }
+
+        // Visit the room
+        cy.visit("/#/room/" + roomId);
+
+        // Wait until configuration is finished
+        cy.contains(
+            ".mx_RoomView_body .mx_GenericEventListSummary[data-layout=" +
+                layout.toLowerCase() + // Ensure GELS (generic event list summary) is specified with the layout variable
+                "] .mx_GenericEventListSummary_summary",
+            "created and configured the room.",
+        ).should("exist");
+    };
+
```

Comment:
I'm a big fan of a helper function for repetition in tests and this looks really good.

A small suggestion that I think might make it neater for people to use in the future, is do you think you'd be able to make the type of the argument use the `Layout` enum that's available in `src/settings/enums/Layout.ts`?

I think it would increase the ease of use and avoid the potential for typos when using this util.